docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+MongoDB:
```

Comment:
[ Improvement ] - I'm wondering if we should not set another port. For people already having a mongo db instance running on 27017, we will encounter conflicts. 

Moreover, I would put this on the test folder to keep consistency over our directory structure. 

Finally I don't know if it's possible, but adding the docker-compose command directly in our tests suit would be a major improvement (no need to run it on our own, entirely handled by tests scripts ...). But I don't know if it's possible.

Reply:
This is just for CI, not for devs.

Reply:
How do we locally run tests so ?